### ISSUE TITLE
HeNeAr line solutions using line list hash table?

### ISSUE THREAD

**Issue by jradavenport** [jradavenport/spectra#6]
can we do a pattern fit to calibration lamp spectra using line lists?

like a 1-d, kindergarten version of astrometry.net solution ala @davidwhogg & @dstndstn ?

**Comment by davidwhogg** [comment]
Yes for sure. I think you take sets of three nearby lines and compute the separation ratio (which varies from 0 to 1 and use that as your geometric hash.

**Comment by jradavenport** [comment]
This was thought up [~20years ago for IRAF](http://adsabs.harvard.edu/abs/1996ASPC..101...33V). It was a good idea then, and still is now!

**Comment by jradavenport** [comment]
I started cracking at this, but its just a mess... 
The beginnings are in the file [linehash.py](https://github.com/jradavenport/pydis/blob/master/pydis/linehash.py)

@migueldvb you might be interested
